### PR TITLE
[DevOps-106] - Create one ingress resource per environment to fix 401

### PR DESCRIPTION
**Pull request by rach-tech**
## Purpose
Initially the ingress object was created with 3 host to direct to their respective service's however the way ingress is handling multi-host seems to be conflicting or some reason causing 401

## Approach
Created ingress resource per environment to avoid these conflicts which resolved the issue.
## Testing

Checklist:

* [x] I have bumped the chart version.
* [ ] Any new values are backwards compatible and/or have sensible default.

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

File: charts/vertx/values-production.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ service:
   name: vertx-svc-production
   type: NodePort
```
Comment (admiraladmirable) [inline charts/vertx/values-production.yaml:47]: This doens't need to be done now, but one optimization we can implement is Container native load balancing. 

This way, we can use `type: ClusterIP` and route directly into the container: https://cloud.google.com/kubernetes-engine/docs/how-to/container-native-load-balancing#create_service

**Review by admiraladmirable — APPROVED** [review]

File: charts/vertx/values-production.yaml [hunk 2]
```diff
@@ -94,7 +93,7 @@ server:
     domain: vertx.topl.services
     # -- Certificate Subject Alternate Names (SANs)
     additionalHosts: []
-    #  - "dev.vertx.topl.services"
+    #  - "beta.vertx.topl.services"
```
[the remark this answers is not in the document]
Comment (rach-tech) [reply charts/vertx/values-production.yaml:96]: Ahh no, I had them to test it out. But removed the commented code now.

File: charts/vertx/values-production.yaml [hunk 3]
```diff
@@ -69,8 +69,7 @@ configMap:
   create: true
   name: vertx-config-production
   data:
-    PORT: 2378
-    HEALTH_CHECK_PORT: 8102
+    PORT: 2380
```
[the remark this answers is not in the document]
Comment (rach-tech) [reply charts/vertx/values-production.yaml:72]: They were different before but had conflict with the network endpoint group hence changed it back to use same port

File: charts/vertx/values-production.yaml (hunk 1 above)
Comment (admiraladmirable) [reply charts/vertx/values-production.yaml:47]: @rach-tech if you're testing this, make sure to add the annotation to the services: 

```
apiVersion: v1
kind: Service
metadata:
  name: neg-demo-svc # Name of Service
  annotations:
    cloud.google.com/neg: '{"ingress": true}' # Creates a NEG after an Ingress is created

```

Comment (rach-tech) [reply charts/vertx/values-production.yaml]: All updated! I think this annotation was already present before but now I updated it in the helm-charts @admiraladmirable

**Pull request merged**